x-pack/plugins/metrics_entities/server/scripts/check_env_variables.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+#
```

Comment:
Missing last blurb in license:


```
/*
 * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
 * or more contributor license agreements. Licensed under the Elastic License
 * 2.0 and the Server Side Public License, v 1; you may not use this file except
 * in compliance with, at your election, the Elastic License 2.0 or the Server
 * Side Public License, v 1.
 */
```

Reply:
Thanks, I will for sure fix these here and resolve it in this PR directly. Appreciate the 🦅 👁️ 's on this.